unleash-yggdrasil/src/strategy_grammar.pest

```diff
@@ -79,17 +81,16 @@ constraint = {
     numeric_constraint = { context_value ~ ordinal_operation ~ num }
     semver_constraint = { context_value ~ ordinal_operation ~ semver }
     rollout_constraint = { percentage ~ stickiness_param? ~ group_id_param?}
-        stickiness_param = { "sticky on " ~ context_value }
+        stickiness_param = { "sticky on " ~ context_value ~ ( NULL_COALESCE ~ context_value)* }
```

Comment:
This is the only important change here (everything else is QoL), this allows us to express things like `sticky on user_id | session_id | random`